### PR TITLE
ENG-729 attributes list options for attribute type list fix

### PR DESCRIPTION
**Pull request by jeffgo10**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **gidesan**

If they are not already assigned, you can assign the PR to them by writing `/assign @gidesan` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-cms).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-cms/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/state/content-type/selectors.js [hunk 1]
```diff
@@ -129,8 +135,10 @@ export const getSelectedCompositeAttributes = createSelector(
 );
 
 const getList = (type, list) => {
+  console.log('getList', type);
```
Comment (ichalagashvili) [inline src/state/content-type/selectors.js:138]: remove this please

File: src/state/content-type/selectors.js [hunk 2]
```diff
@@ -129,8 +135,10 @@ export const getSelectedCompositeAttributes = createSelector(
...
   switch (type) {
     case TYPE_LIST:
+      console.log('what', list.filter(f => !NO_ATTRIBUTE_FOR_TYPE_LIST.includes(f)));
```
Comment (ichalagashvili) [inline src/state/content-type/selectors.js:141]: and this

File: src/state/content-type/selectors.js [hunk 3]
```diff
@@ -151,6 +159,7 @@ export const getContentTypeAttributesIdList = createSelector(
       return getList(code, list);
     }
     const { type } = attributeSelected;
+    console.log('oh its', type);
```
Comment (ichalagashvili) [inline src/state/content-type/selectors.js:162]: this too

**Comment by jeffgo10** [comment]
oops sorry. should be okay now

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/entando-cms/pull/221#pullrequestreview-440870991" title="Approved">ichalagashvili</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-cms).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/entando-cms/blob/master/OWNERS)~~ [ichalagashvili]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**